pkg/helmcertifier/checks/checks_test.go

```diff
@@ -30,34 +31,28 @@ func TestIsHelmV3(t *testing.T) {
 	}
 
 	positiveTestCases := []testCase{
-		{description: "valid tarball, absolute path", uri: "/tmp/chart-v3.tgz"},
-		{description: "valid tarball, relative path", uri: "./chart-v3.tgz"},
-		{description: "valid tarball, http", uri: "http://www.example.com/chart-v3.tgz"},
-		{description: "valid directory, absolute path", uri: "/tmp/chart-v3"},
-		{description: "valid directory, relative path", uri: "./chart-v3"},
+		{description: "valid tarball, absolute path", uri: "testchart-0.1.0.tgz"},
```

Comment:
The intention of those tests were to be used to guide the development, being either part of this changelist or be picked up by someone else, so please revert the change and amend the proper test case.

Reply:
Verifying various kinds of URI patterns need not be part of every check. A separate test would be sufficient for the same.

Reply:
I do agree though that we should not keep around multiple copies of the same test artifact, so let's keep all the test charts as directories in the project, where we could use to at least add test cases for absolute and relative paths; tooling to load remote charts and how to build those tarballs for testing purposes could be postponed for now.

Checks should be treated as black boxes, and integration tests for all inputs are, in my opinion, required.

If you feel this shouldn't be required for every check that loads a chart, then move the tests to scope of `loadChartFromURI` to be implemented there (assuming every check uses `loadChartFromURI` to load a chart, but until that happens:

* Have an unpacked version of the chart inside `$projectRoot/tests/charts` or an equivalent directory, resulting in athe following path: `$projectRoot/tests/charts/chart-0.1.0-v3.valid`
* Tests should change the current working directory to `$projectRoot/tests/charts`
* Relative tests will use `./chart-0.1.0-v3.valid`
* Absolute tests will use `$projectRoot/tests/charts/chart-0.1.0-v3.valid`

With this in place, later it could be possible to create tarballs directly from the chart directories.

Reply:
Another pro of having the chart in a directory is it makes easier to inspect what is inside the chart, and not as an opaque resource as it is with a tarball.

Reply:
Can we do this in a separate PR? I am not sure if all these suggestions need to be a blocker for this PR.

Reply:
Sure; I've started working on #5 that is related.